packages/core/src/service/services/product-option-group.service.ts

```diff
@@ -95,6 +100,21 @@ export class ProductOptionGroupService {
             .then(groups => groups.map(group => this.translator.translate(group, ctx, ['options'])));
     }
 
+    getOptionGroupForFieldResolver(
```

Comment:
I don't really like this naming - there is no other service method named like this and the name does not give you any information about how it differs from `findOne()`.

Since this method is almost identical to `findOne()`, just without the deletedAt clause, maybe we should just add a fourth options arg to `findOne()` that allows you to specify `includeSoftDeleted: boolean`.